http/http_client.hpp

```diff
@@ -206,75 +217,87 @@ class HttpClient : public std::enable_shared_from_this<HttpClient>
+                                     << self->port
+                                     << " for Event: " << eventData["Id"];
+                    self->state = ConnState::recvFailed;
+                    self->handleConnState();
+                    return;
+                }
 
-            BMCWEB_LOG_DEBUG << "recvMessage() bytes transferred: "
-                             << bytesTransferred;
-            boost::ignore_unused(bytesTransferred);
+                BMCWEB_LOG_DEBUG << "recvMessage() bytes transferred: "
+                                 << bytesTransferred;
+                boost::ignore_unused(bytesTransferred);
 
-            // Check if the response and header are received
-            if (!self->parser->is_done())
-            {
-                // The parser failed to receive the response
-                BMCWEB_LOG_ERROR
-                    << "recvMessage() parser failed to receive response"
-                    << " from subId: " << self->subId;
-                self->state = ConnState::recvFailed;
-                self->handleConnState();
-                return;
-            }
+                // Check if the response and header are received
+                if (!self->parser->is_done())
+                {
+                    // The parser failed to receive the response
+                    BMCWEB_LOG_ERROR
+                        << "recvMessage() parser failed to receive response"
+                        << " from subId: " << self->subId
+                        << " Destination: " << self->host << ":" << self->port
+                        << " for Event: " << eventData["Id"];
+                    self->state = ConnState::recvFailed;
+                    self->handleConnState();
+                    return;
+                }
 
-            unsigned int respCode = self->parser->get().result_int();
-            BMCWEB_LOG_DEBUG << "recvMessage() Header Response Code: "
-                             << respCode;
+                unsigned int respCode = self->parser->get().result_int();
+                BMCWEB_LOG_DEBUG << "recvMessage() Header Response Code: "
+                                 << respCode;
 
-            // 2XX response is considered to be successful
-            if ((respCode < 200) || (respCode >= 300))
-            {
-                // The listener failed to receive the Sent-Event
-                BMCWEB_LOG_ERROR << "recvMessage() Listener Failed to "
-                                    "receive Sent-Event. Header Response Code: "
-                                 << respCode << " from subId: " << self->subId;
-                self->state = ConnState::recvFailed;
-                self->handleConnState();
-                return;
-            }
+                // 2XX response is considered to be successful
+                if ((respCode < 200) || (respCode >= 300))
+                {
+                    // The listener failed to receive the Sent-Event
+                    BMCWEB_LOG_ERROR
```

Comment:
I guess this was here before, but why log this?

Reply:
This is to track if we receive any 4xx or 5xx errors in the response header